src/relayclient/RelayProvider.ts

```diff
@@ -83,7 +83,7 @@ export class RelayProvider implements HttpProvider {
     if (this._useGSN(payload)) {
       if (payload.method === 'eth_sendTransaction') {
         if (payload.params[0].to === undefined) {
-          throw new Error('GSN cannot relay contract deployment transactions. Add {from: accountWithEther, useGSN: false}.')
+          throw new Error('GSN cannot relay contract deployment transactions. Add {from: accountWithRBTC, useGSN: false}.')
```

Comment:
Should say RIF Enveloping, or Relay Provider instead of GSN. 
Should change useGSN to useEnveloping

Reply:
This change will be handled in another PR